client/src/components/feedback/edit-question-for-queue-modal.tsx

```diff
@@ -0,0 +1,271 @@
+      name: UtteranceName.NONE,
+      clientId: uuid(),
+      paraphrases: [],
+      mentor: mentor._id,
+    };
+    // create newSubjectQuestion : to add to db
+    const emptyTopic: Topic = { id: "", name: "", description: "" };
+    const newSubjectQuestion: SubjectQuestionGQL = {
+      question: newQuestion,
+      topics: selectedTopic != undefined ? [selectedTopic] : [emptyTopic],
+      category: selectedCategory,
+    };
+    // add to DB
+    addOrUpdateSubjectQuestions(
+      selectedSubject?._id || "",
+      [newSubjectQuestion],
+      accessToken
+    );
+    // add to record queue
+    addQuestionToRecordQueue(newQuestion._id, accessToken);
+    // close modal
+    if (handleClose != undefined) {
+      handleClose();
+    }
+  }
+
+  return (
+    <div>
+      <div>
+        <div className="modal-wrapper">
+          <Modal
+            aria-labelledby="transition-modal-title"
+            data-cy="create-question-modal"
+            aria-describedby="transition-modal-description"
+            className={classes.modal}
+            open={open}
+            onClose={okButtonClicked}
+            closeAfterTransition
+            BackdropComponent={Backdrop}
+            BackdropProps={{
+              timeout: 500,
+            }}
+          >
+            <Fade in={open}>
+              <div className={classes.paper}>
+                <Grid item alignItems="center" xs={12} md={12}>
+                  <TextField
+                    // user-asked question (editable)
+                    label="Create new question:"
+                    required={true}
+                    value={customQuestion} // fill in with OG user question
+                    // create a new question for mentor with (un)edited string
+                    onChange={(e) =>
+                      onTextInputChanged(e, () => {
+                        setCustomQuestion(e.target.value);
+                      })
+                    }
+                    className={classes.inputField}
+                  />
+                  <div
+                    className={classes.inputField}
+                    style={{ textAlign: "left" }}
+                  >
+                    <FormControl>
+                      <InputLabel>Subject</InputLabel>
+                      <Select
+                        // a dropdown for the subject
+                        required={true}
+                        data-cy="subject-drop-down"
+                        style={{ width: 200 }}
+                        onChange={(
+                          event: React.ChangeEvent<{ value: unknown }>
+                        ) => {
+                          setSelectedSubject(
+                            mentor.subjects.find((a) => {
+                              return a._id === event.target.value;
+                            })
+                          );
+                        }}
+                      >
+                        <option value={undefined} selected disabled>
```

Comment:
do you need to explicitly specify that value={undefined}? You can probably just remove that (I see that's what you did below), you usually want to avoid setting this explicitly to undefined